### PR TITLE
Ap customcop

### PR DESCRIPTION
**Pull request by adip1818**
Including the custom cop for LoggerFormat.

### CODE REVIEW

File: lib/cops/logger_format_cop.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+require 'byebug'
```
Comment (SowmyaAmagi) [inline lib/cops/logger_format_cop.rb]: This can be removed

File: lib/cops/logger_format_cop.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+module RuboCop
+  module Cop
+    module CustomCops
+      class LoggerFormatCop < Cop
```
Comment (SowmyaAmagi) [inline lib/cops/logger_format_cop.rb]:
```suggestion
      class PutLoggerFormatCop < Cop
```

File: rubocop.yml [hunk 3]
```diff
@@ -8,7 +8,12 @@ AllCops:
     - 'node_modules/**/*'
 
 require:
+  - './lib/cops/logger_format_cop.rb'
   - 'rubocop-rails'
+  
+
+CustomCops/LoggerFormatCop:
+  Enabled: true
```
Comment (SowmyaAmagi) [inline rubocop.yml]: Lets disable it by default
Enable it in required repo

Comment (adip1818) [reply rubocop.yml]: Okay

**Comment by SowmyaAmagi** [comment]
@adip1818 Can you add a doc explaining about this Cop

**Review by SowmyaAmagi — COMMENTED** [review]
LGTM

**Review by rparjun — APPROVED** [review]
Looks good!

**Review by SowmyaAmagi — APPROVED** [review]

**Pull request merged**